### PR TITLE
🌳 Add cross-domain examples (climate modelling, energy consumption, ...)

### PR DESCRIPTION
**Pull request by jvdd**

### CODE REVIEW

**Comment by jonasvdd** [comment]
LGTM, but still to do:

- [ ] new release where we fix `tsflex.integrations`

**Comment by jonasvdd** [comment]
@jvdd, will you take a look at my commits?

**Comment by jonasvdd** [comment]
Will you look into a new release (where the `make_robust` is moved @jvdd?

File: tsflex/chunking/chunking.py [hunk 1]
```diff
@@ -32,14 +32,14 @@ def chunk_data(
 
     Notes
     -----
-    * As the `fs_dict` is requested, the assumption is made that **each item** in `data`
-      has a **fixed sample frequency** and that **each item** in `data` 
+    * when you set `fs_dict`, the assumption is made that **each item** in `data`
```
Comment (jvdd) [inline tsflex/chunking/chunking.py:35]:
```suggestion
    * When you set `fs_dict`, the assumption is made that **each item** in `data`
```

File: tsflex/chunking/chunking.py [hunk 2]
```diff
@@ -48,9 +48,9 @@ def chunk_data(
         `pd.DatetimeIndex`. 
         The assumption is made that each `item` in data has a _nearly-constant_ sample 
         frequency (when there are no gaps).
-    fs_dict: Dict[str, int]
-        The sample frequency dict. This dict must at least withhold all the keys
-        from the items in `data`.
+    fs_dict: Dict[str, int], optional
+        The sample frequency dict. If set, this dict must at least withhold all the keys
+        from the items in `data`.
```
Comment (jvdd) [inline tsflex/chunking/chunking.py:53]:
```suggestion
        from the items in `data`.
```

File: tsflex/chunking/chunking.py [hunk 3]
```diff
@@ -94,17 +94,31 @@ def chunk_data(
         max_chunk_dur = parse_time_arg(max_chunk_dur)
     sub_chunk_overlap = parse_time_arg(sub_chunk_overlap)
 
-    # Assert that there are no duplicate series names and the names reside in fs_dict
+    # Assert that there are no duplicate series names
     assert len(series_list) == len(set([s.name for s in series_list]))
```
Comment (jvdd) [inline tsflex/chunking/chunking.py:98]:
```suggestion
    assert len(series_list) == len(set(s.name for s in series_list))
```

File: tsflex/chunking/chunking.py [hunk 4]
```diff
@@ -94,17 +94,31 @@ def chunk_data(
...
-    assert all([str(s.name) in fs_dict for s in series_list])
 
     # Default arg -> set the chunk range margin to 2x the min-freq its period
     if chunk_range_margin is None:
-        chunk_range_margin = 2 / min([fs_dict[str(s.name)] for s in series_list])
+        if fs_dict is not None:
+            chunk_range_margin = 2 / min([fs_dict[str(s.name)] for s in series_list])
```
Comment (jvdd) [inline tsflex/chunking/chunking.py:103]:
```suggestion
            chunk_range_margin = 2 / min(fs_dict[str(s.name)] for s in series_list)
```

**Comment by jvdd** [comment]
> @jvdd, will you take a look at my commits?

LGTM!

**Pull request merged**